website/docs/faq.md

```diff
@@ -160,6 +160,90 @@ out-of-bounds. If you would rather raise an exception than handle `nil`, use the
 [0, 1, 2].fetch(3) # IndexError: index 3 outside of array bounds
 ```
 
+## Sigs are vague for std lib methods that accept keyword arguments & have multiple return types
```

Comment:
I think we generally use `stdlib` or `standard library` see [here](https://sorbet.org/docs/faq#it-looks-like-sorbets-types-for-the-stdlib-are-wrong) for example.

```suggestion
## Sigs are vague for stdlib methods that accept keyword arguments & have multiple return types
```